tests/src/test/fdc3.raiseIntent.ts

```diff
@@ -168,3 +175,17 @@ export default () =>
       }
     });
   });
+
+const validateIntentResolution = (
+  appName: string,
+  intentResolution: IntentResolution
+) => {
+  if (typeof intentResolution.source === "string") {
+    expect(intentResolution.source).to.eq(`${appName}Id`, raiseIntentDocs);
```

Comment:
When `source` is a string it's expected to be the `name` rather than `appId`, see: https://fdc3.finos.org/docs/1.2/api/ref/Types#targetapp

This changes in 2.0 to always be an `AppIdentifier` (as `name` is deprecated wherever used and the `TargetApp` type was dropped in favour of `source` always being an `AppIdentifier` with `appId`).

Reply:
```suggestion
    expect(intentResolution.source).to.eq(appName, raiseIntentDocs);
```